examples/correlation_matrices/sampler.cpp

```diff
@@ -89,6 +89,26 @@ void write_to_file(std::string filename, std::vector<PointType> const& randPoint
     std::cout.rdbuf(coutbuf);
 }
 
+bool is_correlation_matrix(const MT& matrix){
+
+	const double tol = 1e-8;
```

Comment:
this could be a function parameter with `1e-8` the default value.

Reply:
i updated this